apps/src/blockly/customBlocks/googleBlockly/behaviorBlocks.js

```diff
@@ -184,26 +184,11 @@ GoogleBlockly.Extensions.registerMutator(
  * @param {WorkspaceSvg} workspace The workspace containing procedures.
  * @returns an array of XML block elements
  */
-export function flyoutCategory(workspace, functionEditorOpen = false) {
+export function flyoutCategory(workspace) {
   const blockList = [];
 
-  const behaviorDefinitionBlock = {
```

Comment:
The modal editor knows how to make a new behavior definition block without this. This was previously used to add a def block to the flyout if the modal editor was disabled, but we do not do that any more.